docs/3.1/docs/canton/reference/error_codes.rst

```diff
@@ -320,20 +320,13 @@ in a sensible way to automatically deal with errors and decide whether to retry
 a request or escalate to the operator.
 
 ..
-   This file is generated:
-.. include:: error-categories-inventory.rst.inc
-
-
-Error Codes Inventory - Daml
-****************************
-
-..
-   This file is generated:
-.. include:: error-codes-inventory.rst.inc
+   This file is manually generated using ErrorCategoryInventoryDocsGenApp the checked in
+   This was done as the daml variant is not maintained and due for removal
+.. include:: ./canton-error-categories-inventory.rst.inc
```

Comment:
I'm not clear on the comment.

Does `ErrorCategoryInventoryDocsGenApp` still exist? And do we need to continue to use it to generate the content for `canton-error-categories-inventory.rst.inc`?

If so, see my general comment about checking in generated content.

Reply:
Yes. My original proposal was to add this to the canton distribution and use this file with docs. I was told to hold off doing that for a few weeks as it was/is apparently unclear if the docs.daml.com repo will be merged back into the canton repo.

Once there is clarity regarding the above it should be done and I've raised an issue (https://github.com/DACH-NY/canton/issues/18872) to track this.

On the 2.9 branch this cannot be so easily done as these error codes are referenced by the ledger-api-test-tool (that in 3.x has been moved to the canton repo).